assets/stylesheets/sections/media.scss

```diff
@@ -0,0 +1,15 @@
+/** @format */
+
+@import '../shared/mixins/breakpoints';
+@import '../shared/mixins/clear-text';
+@import '../shared/mixins/heading';
+@import '../shared/mixins/long-content-fade';
+@import '../shared/mixins/noticon';
+
```

Comment:
We're probably missing an import for the `z-index` function here since the `media.css` file generated contains rules like this:

```javascript
z-index: z-index("screen-reader-text-parent", ".screen-reader-text:focus");
```